README.md

```diff
@@ -80,139 +97,7 @@ cd ldes-server-application
-| ingest.port                | Determines the port on which the ingest api will be available                                                       | No       | value of server.port | 8080                  | any available port number |
-
-##### Example Serving Static Content
-
-  ```yaml
-spring:
-  mvc:
-    static-path-pattern: { pattern used in url for static content, e.g. /content/** for serving on http://localhost:8080/content/ }
-  web:
-    resources:
-      static-locations: { source folder of static content, e.g. file:/opt/files }
-  ```
-
-##### Example Serving DCAT Metadata
-
-Supported file formats: .ttl, .rdf, .nq and .jsonld
-Templates for configuring the DCAT metadata can be found [here](templates/dcat)
-A detailed explanation on how to manage and retrieve the DCAT metadata can be
-found [here](ldes-server-admin/README.md#dcat-endpoints).
-
-##### Relative urls
-
-To enable relative urls on the server, set the following property:
-
-  ```yaml
-ldes-server:
-  use-relative-url: true
-  ```
-
-When fetching any page using relative urls, any resource hosted on the server will have a URI relative to the requested
-page.
-
-> **Note**: When using relative urls, GET requests can not be performed with N-quads or triples as accept-type.
-> This is because these types don't support resolving the relative URI's
-
-##### Port bindings
-
-To change the ports of the diffrent API's, use the following config:
-  ```yaml
-ldes-server:
-  admin:
-    port: 8080
-  fetch:
-    port: 8081
-  ingest:
-    port: 8082
-  ```
-
-Any combination of these ports can be the same of completely omitted.
-When no port number is given, the default server port will be used.
-
-### Docker Setup
-
-#### Docker-compose
-
-There are 2 files where you can configure the dockerized application:
-
-- [The config files](#the-config-files)
-- [The docker compose file](#the-docker-compose-file)
-
-#### The Config Files
-
-Runtime settings can be defined in the configuration files. Use [config.env](docker-compose/config.env) for a public
-setup and be sure not to commit this file, as it contains secrets. For a local setup,
-use [config.local.env](docker-compose/config.local.env).
-
-#### The docker compose File
-
-Change the `env_file` to `config.env` or `config.local.env` in [compose.yml](docker-compose.yml) according to
-your needs.
-
-#### Starting the Dockerized Application
-
-Run the following commands to start the containers:
-
-```bash
-COMPOSE_DOCKER_CLI_BUILD=1 docker-compose build
-docker-compose up
-```
-
-> **Note**: Using Docker Desktop might fail because of incorrect environment variable interpretation.
->
-> ```unexpected character "-" in variable name near ...```
+The needed profiles can be found in the Structure of the application section.
```

Comment:
Maybe one example with only one module loaded?

Reply:
purposefully not doing it to not make our docs deprecated soon :)